freeipa.spec.in

```diff
@@ -1025,6 +1025,15 @@ if [ -e /usr/sbin/ipa_kpasswd ]; then
 # END
 fi
 
+# create users and groups
+# create kdcproxy group and user
+getent group kdcproxy >/dev/null || groupadd -f -r kdcproxy
+getent passwd kdcproxy >/dev/null || useradd -r -g kdcproxy -s /sbin/nologin -d / -c "IPA KDC Proxy User" kdcproxy
```

Comment:
Not now. When (and more importantly if) we get uid and gid assigned I will update the code. Now I don't have any idea what value I could use.
Also looking into related ticket https://pagure.io/packaging-committee/issue/474 (opened by you by the way) it seems that "we run in container and store data to volume" is not enough to allocate static uid and gid.